app/src/main/res/values/styles.xml

```diff
@@ -77,4 +77,10 @@
         <item name="android:layout_height">match_parent</item>
         <item name="android:fitsSystemWindows">true</item>
     </style>
+
+    <style name="SocialMediaButton" parent="Theme.AppCompat.Light">
```

Comment:
Thanks 👍 
I think Theme.AppCompat.Light is a theme. So we should not use it for Button. What do you think?

Reply:
I agree with you. This is a mistake. I'll fix this soon!

Reply:
Can you fix this?

Reply:
Oops, I forgot fix this... I'm sorry to have a trouble many times... 😢